types/jquery_countdown/tslint.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
```

Comment:
I'd prefer you didn't disable these lint rules.

Reply:
What lint rules concern you? Things like max-line-length and one-line are more of a coding preference, but I can change them if that is a DefinitelyTyped standard.

Reply:
I looked through the contribution guide and found that there is [no coding style](http://definitelytyped.org/guides/contributing.html#code-style). These are in place so that I don't get error when I run the linter. Do you know where the standard you are referring to is documented?

What lint rules are you concerned about and why? If you have a good reason to change something I will be happy to make the change. I just need a what and why.

Reply:
There is no uniform style across the entire project because the project began long before we setup the linter. However we try to keep new definitions consistent with a uniform set of lint rules (the base config), in order to make maintenance easier (it helps remove subjectivity from the formatting discussion). Many of the rules are there to prevent common mistakes. `ban-types`, for instance prevents you from using `String` or `Object` (the interfaces), because their presence is almost always incorrect, as they should likely be `string` or `object` (the primitives), instead. This can be super important, since  your or I may not be the next person to edit this file, and the next editor may lack context or experience. The linter passing greatly expedites code reviews, since many checks are done automatically. Generally, if a rule must be worked around in a specific location to accomplish a specific goal, an inline suppression is used (this way the nonconformity is called out during review and can be properly audited by reviewers).